src/lib/output/themes/default/partials/icon.tsx

```diff
@@ -146,28 +211,52 @@ export const icons: Record<
     },
     [ReflectionKind.Property]: () =>
         kindIcon(
-            <path
-                d="M9.354 16V7.24H12.174C12.99 7.24 13.638 7.476 14.118 7.948C14.606 8.412 14.85 9.036 14.85 9.82C14.85 10.604 14.606 11.232 14.118 11.704C13.638 12.168 12.99 12.4 12.174 12.4H10.434V16H9.354ZM10.434 11.428H12.174C12.646 11.428 13.022 11.284 13.302 10.996C13.59 10.7 13.734 10.308 13.734 9.82C13.734 9.324 13.59 8.932 13.302 8.644C13.022 8.356 12.646 8.212 12.174 8.212H10.434V11.428Z"
-                fill="var(--color-text)"
-            />,
-            "#FF984D",
+            <text
+                fill="var(--color-icon-text)"
+                font-family="var(--icon-font-family)"
+                font-size="var(--icon-font-size)"
+                font-weight="var(--icon-font-weight)"
+                font-style="var(--icon-font-style)"
+                x="7.5"
+                y="18"
+            >
+                P
+            </text>,
+            "var(--color-ts-property)",
             true,
         ),
     [ReflectionKind.Reference]: () =>
         kindIcon(
-            <path
-                d="M10.354 17V8.24H13.066C13.586 8.24 14.042 8.348 14.434 8.564C14.826 8.772 15.13 9.064 15.346 9.44C15.562 9.816 15.67 10.256 15.67 10.76C15.67 11.352 15.514 11.86 15.202 12.284C14.898 12.708 14.482 13 13.954 13.16L15.79 17H14.518L12.838 13.28H11.434V17H10.354ZM11.434 12.308H13.066C13.514 12.308 13.874 12.168 14.146 11.888C14.418 11.6 14.554 11.224 14.554 10.76C14.554 10.288 14.418 9.912 14.146 9.632C13.874 9.352 13.514 9.212 13.066 9.212H11.434V12.308Z"
-                fill="var(--color-text)"
-            />,
-            "#FF4D82", // extract into a CSS variable potentially?
+            <text
+                fill="var(--color-icon-text)"
+                font-family="var(--icon-font-family)"
+                font-size="var(--icon-font-size)"
+                font-weight="var(--icon-font-weight)"
+                font-style="var(--icon-font-style)"
+                x="7"
+                y="17.5"
+            >
+                R
+            </text>,
+            "var(--color-ts-reference)",
             true,
         ),
     [ReflectionKind.SetSignature]() {
         return this[ReflectionKind.Accessor]();
     },
     [ReflectionKind.TypeAlias]: () =>
         kindIcon(
-            <path d="M11.31 16V8.224H8.91V7.24H14.79V8.224H12.39V16H11.31Z" fill="var(--color-text)" />,
+            <text
+                fill="var(--color-icon-text)"
+                font-family="var(--icon-font-family)"
+                font-size="var(--icon-font-size)"
+                font-weight="var(--icon-font-weight)"
+                font-style="var(--icon-font-style)"
+                x="7"
+                y="17.5"
+            >
+                T
```

Comment:
This doesn't appear quite centered

![image](https://github.com/user-attachments/assets/e4470276-e39f-454a-b3f1-a19ec52ef06f)